src/main/resources/LabelsBundle_cy.properties

```diff
@@ -40,6 +40,7 @@ news=Newyddion
 our-offices=Ein swyddfeydd
 connect-with-us=Cysylltu â ni
 twitter=Twitter
+instagram=Instagram
```

Comment:
Although I can see the other social media labels are duplicated, I don't think we need to add it twice?